src/components/Popup.js

```diff
@@ -168,25 +171,36 @@ function Title(props) {
 
     popupParent.style.top = newPosition.y + 'px';
     popupParent.style.left = newPosition.x + 'px';
-  });
+
+    // notify interested parties
+    isFunction(emit) && emit('dragover', { newPosition, delta });
```

Comment:
Probably a left over:

```suggestion
    emit('dragover', { newPosition, delta });
```

Reply:
Thanks 👍